src/index.js

```diff
@@ -1,12 +1,19 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import App from "./App";
+import Footer from "./components/Footer";
+import Header from "./components/Header";
+import Main from "./components/Main";
 import "./index.css";
+// import Main from "./Main";
 import reportWebVitals from "./reportWebVitals";
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
+    <Header />
+    <Footer />
+    <Main />
   </React.StrictMode>,
   document.getElementById("root")
 );
```

Comment:
Good work on the lab so far!

I have set up apps like this before. Although, I believe 'index.js' is like the entry point file. I believe it is best practice to keep our main application/ components in `App.js.` I am not 100% certain on this, but I think this is the case. Read here: [React JS Apps](https://www.freecodecamp.org/news/quick-guide-to-understanding-and-creating-reactjs-apps-8457ee8f7123/) for more info on index.js and App.js.

Instead of adding in our components here, we can import them into App.js 👍